client/src/arcplanner/lib/src/util/databaseHelper.dart

```diff
@@ -116,13 +119,14 @@ class DatabaseHelper {
           $_taskTID TEXT PRIMARY KEY CHECK(LENGTH($_taskTID) = $_uuidSize), 
           $_taskTitle TEXT CHECK(LENGTH($_taskTitle) <= $_nameSize), 
           $_taskDesc TEXT, 
-          $_taskDueDate TEXT, 
+          $_taskDueDate TEXT,
+          $_taskTimeDue TEXT,
           $_taskLoc TEXT CHECK(LENGTH($_taskLoc) <= $_locSize),
           $_taskCompleted INTEGER CHECK($_taskCompleted == 0 OR $_taskCompleted == 1)
           )""");
     await db.execute("""
         CREATE VIEW $_arcView AS
-        SELECT UID, AID, Title, Description, DueDate, ParentArc, Completed, 
+        SELECT UID, AID, Title, Description, DueDate, TimeDue ParentArc, Completed, 
```

Comment:
Should be a comma between TimeDue and ParentArc. Does not affect the operation of the DB, but for consistency.